### DISCUSSION TITLE
这张Tensorboard图应该怎么看是否可以停止训练了

### DISCUSSION THREAD

**Discussion by FuSeven** [svc-develop-team/so-vits-svc#146] [General]
<img width="1229" alt="屏幕截图 2023-04-12 024728" src="https://user-images.githubusercontent.com/130538506/231374186-7473108c-e25a-4370-935d-4786d3bd92c3.png">

里面的几个值应该关注哪些？谢谢

**Comment by Miuzarte** [comment]
数据集长度有多少？？400k有点太多了
至于什么时候停止训练，有底模训练的时候训练后期两个模型推理效果几乎听不出区别就可以了，五小时的数据集跑个100k结束没问题

**Reply by FuSeven** [reply]
> 数据集长度有多少？？400k有点太多了 至于什么时候停止训练，有底模训练的时候训练后期两个模型推理效果几乎听不出区别就可以了，五小时的数据集跑个100k结束没问题

数据集长度2小时。谢谢您的解答。
另外这个图哪些项能作为参考？

**Reply by Miuzarte** [reply]
> > 数据集长度有多少？？400k有点太多了 至于什么时候停止训练，有底模训练的时候训练后期两个模型推理效果几乎听不出区别就可以了，五小时的数据集跑个100k结束没问题
> 
> 数据集长度2小时。谢谢您的解答。
> 另外这个图哪些项能作为参考？

有底模的情况下g/mel一般都是在24-20之间（取决于数据质量），降到18应该已经够了

**Reply by FuSeven** [reply]
明白了，谢谢您：）
